Microsoft.DotNet.Interactive.Jupyter/IPubSubChannel.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Jupyter.Protocol;
+
+namespace Microsoft.DotNet.Interactive.Jupyter
+{
+    public interface IPubSubChannel
+    {
+        bool Send(JupyterPubSubContent content, Message request, string ident = null);
+    }
```

Comment:
What's the use case and expectation for ident being null?

Reply:
not always needed to generate the topic